src/strategies/property_readers.sml

```diff
@@ -137,6 +29,20 @@ fun label str = [str];
 fun collection str = if str = "NONE" then []
                      else map stringTrim (String.tokens (fn c => c = #",") str);
 
+(*)
```

Comment:
Why is this commented out? If it is potentially useful in future, uncomment it but leave a comment as to why it is there. If it is a relic of old ideas, please delete it.

Reply:
Done